lib/content-services/.storybook/main.js

```diff
@@ -1,26 +1,15 @@
 const rootMain = require('../../../.storybook/main');
 
-
 module.exports = {
   ...rootMain,
-
-  core: { ...rootMain.core, builder: 'webpack4' },
-
+  core: { ...rootMain.core, builder: 'webpack5' },
   stories: [
     ...rootMain.stories,
     '../**/*.stories.@(js|jsx|ts|tsx)'
   ],
-  addons: [...rootMain.addons ],
-  webpackFinal: async (config, { configType }) => {
-    // apply any global webpack configs that might have been specified in .storybook/main.js
-    if (rootMain.webpackFinal) {
-      config = await rootMain.webpackFinal(config, { configType });
-    }
-
-
-
-    // add your own webpack tweaks if needed
-
-    return config;
-  },
+  staticDirs: [
+    ...rootMain.staticDirs,
+    { from: "../../content-services/src/lib/i18n", to: 'assets/adf-content-services/i18n' }
```

Comment:
I think there should be a way to avoid { from: "../../content-services
something like _dirname?

Reply:
replaced by __dirname